stf/model/serializer/lightgbm/quantile.py

```diff
@@ -0,0 +1,49 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
```

Comment:
As mentoined before, probably best to remove this file

Reply:
I'll change it, thanks!